fedimint-server/src/config/io.rs

```diff
@@ -123,3 +123,19 @@ fn encrypted_json_write<T: Serialize + DeserializeOwned>(
     let bytes = serde_json::to_string(obj)?.into_bytes();
     encrypted_write(bytes, key, path.with_extension(ENCRYPTED_EXT))
 }
+
+pub fn remove_existing_file(path: PathBuf) -> anyhow::Result<()> {
+    if path.exists() {
+        fs::remove_file(path)?;
+    }
+    Ok(())
+}
+
+pub fn remove_server_config(path: PathBuf) -> anyhow::Result<()> {
+    remove_existing_file(path.join(LOCAL_CONFIG).with_extension(JSON_EXT))?;
+    remove_existing_file(path.join(CONSENSUS_CONFIG).with_extension(JSON_EXT))?;
+    remove_existing_file(path.join(CLIENT_INVITE_CODE_FILE))?;
+    remove_existing_file(path.join(CLIENT_CONFIG).with_extension(JSON_EXT))?;
+    remove_existing_file(path.join(PRIVATE_CONFIG).with_extension(ENCRYPTED_EXT))?;
```

Comment:
This is very brittle if we ever add any files. In another PR we should introduce a list of files that FM potentially creates and have a test that checks that data dirs contain only these and nothing more.

Reply:
tracking here #3692